src/main/java/Example1.java

```diff
@@ -3,12 +3,14 @@
 import java.util.Properties;
 
 public class Example1
-  extends Properties
-  implements Serializable
+    extends Properties
+    implements Serializable
```

Comment:
I believe our Eclipse codestyle produces this indentation but is this change consistent with IntelliJ?

Reply:
> I believe our Eclipse codestyle produces this indentation but is this change consistent with IntelliJ?

I'm working on making sure both formatters are consistent with the style.

Reply:
> I'm working on making sure both formatters are consistent with the style.

I think if we're making changes to the codestyle we need to socialize it across the developers.

Reply:
> I think if we're making changes to the codestyle we need to socialize it across the developers.

Hence the pull request.  :-)  But if you think I should stand on top of a mountain somewhere and yell, I can do that too.

Reply:
> Hence the pull request. :-) But if you think I should stand on top of a mountain somewhere and yell, I can do that too.

I doubt most people are watching the repository :)  Perhaps an email to the developer list outlining the desired changes in case people have concerns? With so many people I'm not sure what consensus looks like now.

Reply:
Just speaking for myself, my interest in codestyle/checkstyle is limited (feeling fine with the status quo, favoring other battles etc.) which translates to a limited amount of effort I'm willing to put into reviewing and following these PRs. On the off chance I'm not alone in this regard, it might be tough to get a positive affirmation from a huge group of others devs on your efforts. Instead, you might have to employ a model where all affected folks are notified of the proposed changes, thereby given a chance to raise concerns/objections, and interpret the lack thereof after sufficient time as silent consensus to move forward and at least start the experiment which later on can always be revised if additional voices join.